frontend/web/routes.js

```diff
@@ -158,6 +159,11 @@ export default (
         exact
         component={AuditLogPage}
       />
+      <Route
+        path='/project/:projectId/environment/:environmentId/audit-log/:id'
```

Comment:
I have a feeling for self hosted / single image this might need the trailing slash. I will test and confirm.

Reply:
Yep, tested and confirmed broken. Will fix.

Reply:
Just realising that I think this is a FE route... will find the API call and change it there...